src/algebra/char_p/two.lean

```diff
@@ -43,6 +44,34 @@ variables [ring R] [char_p R 2]
 lemma neg_eq (x : R) : -x = x :=
 by rw [neg_eq_iff_add_eq_zero, ←two_smul R x, two_eq_zero, zero_smul]
 
+lemma one_eq_neg_iff {R} [ring R] [nontrivial R] : ring_char R = 2 ↔ (-1 : R) = 1 :=
```

Comment:
```suggestion
lemma one_eq_neg_iff {R} [ring R] [nontrivial R] : (-1 : R) = 1 ↔ ring_char R = 2 :=
```
or
```suggestion
lemma ring_char_eq_iff_one_eq_neg {R} [ring R] [nontrivial R] : ring_char R = 2 ↔ (-1 : R) = 1 :=
```
so that the name and statement match